test/a11y-wand-test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Tests for a11yTextWand
+ */
+
+let assert = require("assert");
+let mockDom = require("./mock-dom");
+
+describe("Screen Reader Wand", function() {
+    let plugin = null;
+
+    before(function(done) {
+        mockDom.createDom(function() {
+            let ScreenReaderWand = require("../plugins/a11y-text-wand");
+            plugin = new ScreenReaderWand();
+
+            document.body.innerHTML = `
+                <article id="article">
+                    <p>This paragraph is visible.</p>
+                    <p id="p-with-hidden">This paragraph has a <span id="span" aria-hidden="true">hidden</span> span.</p>
+                    <p id="hidden-p" aria-hidden="true">This paragraph is hidden.</p>
+                </article>
+            `;
+
+            done();
+        });
+    });
+
+    it("should handle aria-hidden correctly", function() {
+        plugin.run();
+
+        $(document).trigger("mousemove.wand", $("#span"));
+
+        assert($(".tota11y-info-section").text() === "No text visible to a screen reader");
+    });
+
+    it("should handle nested aria-hidden correctly", function() {
+      plugin.run();
+
+      $(document).trigger("mousemove.wand", $("#article"));
+
+      assert($(".tota11y-info-section").text() === "This paragraph is visible. This paragraph has a span.");
+
+      $(document).trigger("mousemove.wand", $("#p-with-hidden"));
+
+      assert($(".tota11y-info-section").text() === "This paragraph has a span.");
+    });
```

Comment:
Thank you for adding tests! What do you think about reworking this last test into two test cases, one for the first assert and one for the second? I think that way, the test names will more clearly document what outcomes we expect from specific states (like "should not include nested aria-hidden", etc.).

Reply:
Sure thing. I'll update the PR.